docs/source/install/__packages_beta_warning.rst

```diff
@@ -0,0 +1,7 @@
+.. warning::
+
+   Packages based installation is currently is beta and upgrades are not yet
```

Comment:
> `Packages based installation is currently is beta and upgrades are not yet supported.`

This is confusing for me because it's not clear enough what exactly means `upgrades`?
* `upgrades` within new packages for example from `1.3.2` to `1.4.0`? (no problem with that)
* `upgrades` from `AIO` installer to new packages? (it's not clear from that message at all)

Also in case of upgrading from `AIO` to new packages - I think we shouldn't include any info about that at all, because it's just different installation method.
Just big `BETA` notice, +1 with @emedvedev